### ISSUE TITLE
Startup Resolution is modern

### ISSUE THREAD

**Issue by blaberry** [NeotokyoRebuild/neo#148]

**Comment by blaberry** [comment]
Needs first-time install for a set of reproducible steps

**Comment by brysondev** [comment]
> I assume on first time install on windows it's still set to 800x600 or sth?

it can vary. I found my install launches in the lowest 16:9 value in the list. (1176x664)
![image](https://github.com/user-attachments/assets/314b39f9-11e8-42ad-9d61-1963b346ae3a)

**Comment by brysondev** [comment]
Also, launching with `-autoconfig` launches the game with the default config settings set by radi8. We should probably be editing that in order to not have this issue.

**Comment by brysondev** [comment]
> @brysondev how did you test first time installing btw? I would think for a proper "first time install" you'd want to delete all the registry keys relating to video settings for src2013mp or whatever

Yeah I already launched tf2 so I might have already had reg keys or whatever. But as I stated earlier `-autoconfig` should accurately simulate a fresh install, and running it does default my game to 800x600.
